### PR TITLE
Add composer.json file for themes which were missing

### PR DESCRIPTION
**Pull request by RCheesley**
| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | features
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | N/A
| Related user documentation PR URL      | N/A
| Related developer documentation PR URL | N/A
| Issue(s) addressed                     | N/A

#### Description:

All themes and plugins need to have a composer.json file now, but a couple of our newer themes were missing them. This PR adds the composer.json files for those themes.

#### Steps to test this PR:

Mostly this just needs a code review of the composer.json files - at present although the files exist in separate repositories and are synced automatically, we still ship them with core.

### CODE REVIEW

File: themes/confirmme/composer.json [hunk 1]
```diff
@@ -2,7 +2,6 @@
   "name": "mautic/theme-confirmme",
   "description": "Mautic Confirmme Theme",
   "type": "mautic-theme",
-  "version": "1.0",
```
Comment (RCheesley) [inline themes/confirmme/composer.json:5]: This is no longer required, I believe - we are using tags for versioning which will be kept in sync with Mautic release versions.

**Comment by RCheesley** [comment]
I think that Codecov has a problem when the PR does not include any PHP files, perhaps? It seems to be somewhat confused!

File: themes/confirmme/composer.json (hunk 1 above)
Comment (shinde-rahul) [reply themes/confirmme/composer.json:5]: 👍

**Review by shinde-rahul — APPROVED** [review]
Changes look good to me!

**Pull request merged**